pkg/provider/azure_vmss_cache.go

```diff
@@ -280,15 +275,19 @@ func (ss *ScaleSet) DeleteCacheForNode(nodeName string) error {
 		return err
 	}
 
-	err = ss.vmssVMCache.Delete(getVMSSVMCacheKey(node.resourceGroup, node.vmssName))
+	// Delete VMSS cache
+	cached, err := ss.vmssCache.Get(consts.VMSSKey, azcache.CacheReadTypeUnsafe)
```

Comment:
Can you rebase your branch? This is not aligned with the master.

Reply:
Updated.